docs/publishers/cli.md

```diff
@@ -0,0 +1,65 @@
+# DPM: the datapackage manger CLI
+
+The dpm is a command-line tool aimed to help publishers to prepare and upload data to the Datapackage Registry. With dpm you will be able to:
```

Comment:
Data Package not Datapackage - ditto throughout. (So data package not datapackage)

This has not been totally clear before!